templates/publisher/register-snap.html

```diff
@@ -26,7 +26,7 @@ <h1 class="p-heading--two">Register snap</h1>
         <div class="col-8 push-2">
             <div class="p-notification--caution">
               <p class="p-notification__response">
-                Another publisher already registered {{ snap_name }}. You can <a href="https://dashboard.snapcraft.io/register-snap/" class="p-link--external" target="_blank">file a dispute</a> to request a transfer of ownership or register a new name below.
+                Another publisher already registered {{ snap_name }}. You can <a href="/register-name-dispute?snap-name={{snap_name}}" class="p-link--external" target="_blank">file a dispute</a> to request a transfer of ownership or register a new name below.
```

Comment:
`class="p-link--external"` it's not an external link anymore

Reply:
Remove along with `target="_blank"`.